sky/tools/sky_snapshot/main.cc

```diff
@@ -20,6 +20,7 @@
 
 void Usage() {
   std::cerr << "Usage: sky_snapshot"
+            << " --" << switches::kPackages
             << " --" << switches::kPackageRoot << " --" << switches::kSnapshot
```

Comment:
This will cause the usage line to claim to need both --packages and --package-root.
We should probably just drop mention of --package-root.